payloads/library/ftp_exfiltrator/payload.txt

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+#
+# Title:         FTP Exfiltrator
+# Author:        Nutt
+# Version:       1.0
+# Target:        Windows
+# 
+#Exfiltrates files from the users Documents folder
+#FTP's all files/folders to a specified FTP site named by the victim hostname.
+#Powershell FTP script will stay running after BashBunny is unplugged, once light turns green unplug and check FTP site.
+
+#Executes 1.ps1
+
+#Purple.........Setup
+#Red............Failed - Need to work on

```

Comment:
Just a thought, but maybe you could have the power shell script write a file to the Bash Bunny to confirm that the script is running.

For example: 
When the powershell script starts, it places a "started" file on the bash bunny. The bash bunny waits for this file to be created, and if it is created then we know the script started successfully. If the "started" file is not created, say within 10 seconds, then we can assume the powershell script failed to execute and then the LED can be turned to red.

Just a thought though. I don't have a bash bunny so I'm not sure how well that would work in reality.